### ISSUE TITLE
Colab Issue with tensorflow-io

### ISSUE THREAD

**Issue by yongtang** [tensorflow/io#369]
Recently tried with Colab for the first time, and see crashes on certain versions:
```python
#!pip install tensorflow-io-2.0-preview     # <= success
#!pip install tensorflow-io-nightly.        # <= crash
#!pip install tensorflow-io                 # <= crash

!curl -OL http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
!curl -OL http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz

import tensorflow as tf
import tensorflow_io.mnist as mnist_io

# Read MNIST into tf.data.Dataset
d_train = mnist_io.MNISTDataset(
    'train-images-idx3-ubyte.gz',
    'train-labels-idx1-ubyte.gz',
    batch=1)

# By default image data is uint8 so conver to float32.
d_train = d_train.map(lambda x, y: (tf.image.convert_image_dtype(x, tf.float32), y))

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(512, activation=tf.nn.relu),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation=tf.nn.softmax)
])
model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

model.fit(d_train, epochs=5, steps_per_epoch=10000)
```

Don't know if there are any binary incompatibility among those version. TF 2.0 + tensorflow-io seems to be fine (with `tensorflow-io-2.0-preview`)

**Comment by yongtang** [comment]
With the current tensorflow-io-nightly the crash is gone in Colab:
```
!pip install tensorflow-io-nightly

!curl -OL http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
!curl -OL http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz

import tensorflow as tf
import tensorflow_io.mnist as mnist_io

# Read MNIST into tf.data.Dataset
d_train = mnist_io.MNISTDataset(
    'train-images-idx3-ubyte.gz',
    'train-labels-idx1-ubyte.gz',
    batch=1)

# By default image data is uint8 so conver to float32.
d_train = d_train.map(lambda x, y: (tf.image.convert_image_dtype(x, tf.float32), y))

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(512, activation=tf.nn.relu),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation=tf.nn.softmax)
])
model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

model.fit(d_train, epochs=5, steps_per_epoch=10000)
```

I will label this issue as resolved.
